pyproject.toml

```diff
@@ -11,6 +11,16 @@ dependencies = [
     "ai-exchange>=0.9.0",
     "click>=8.1.7",
     "prompt-toolkit>=3.0.47",
+    "mkdocstrings>=0.26.1",
```

Comment:
just noticed these should be dev dependencies.

Reply:
yeah sorry i wasn't sure where to put mkdocs deps -- can you expand on what you mean?